packages/cli/src/lib/index.js

```diff
@@ -190,6 +195,29 @@ export const prettierOptions = () => {
   }
 }
 
+// TODO: Move this into `generateTemplate` when all templates have TS support
+/*
+ * Convert a generated TS template file into JS.
+ */
+export const transformTSToJS = (filename, content) => {
+  const result = babel.transform(content, {
+    filename,
+    configFile: false,
+    plugins: [
+      [
+        '@babel/plugin-transform-typescript',
+        {
+          isTSX: true,
+          allExtensions: true,
+        },
+      ],
+    ],
+    retainLines: true,
```

Comment:
oh I didn't know about this option... I think it'd make for prettier (no pun intended) output, I'll add this to the service and sdl generation (although I'm not touching that PR until it lands at this point)

Reply:
Without it the `transform` command would strip newlines in the template, and `prettify` wouldn't re-add them.